src/main/java/nextstep/courses/domain/Session/PayType.java

```diff
@@ -0,0 +1,19 @@
+package nextstep.courses.domain.Session;
+
+public enum PayType {
+    FREE, PAY;
+
+    public void isValid(long price) {
+        if (isFree() && price > 0) {
+            throw new IllegalArgumentException("무료 강의는 0원이어야 합니다.");
+        }
+
+        if (!isFree() && price == 0) {
```

Comment:
부정문은 상대적으로 가독성이 떨어지니 긍정문으로 표현하면 어떨까요?
```suggestion
        if (isPay() && price == 0) {
```